src/Arcus.Observability.Telemetry.AzureFunctions/Extensions/ILoggerBuilderExtensions.cs

```diff
@@ -15,6 +16,7 @@ public static class ILoggingBuilderExtensions
         /// except the specific Azure Functions registrations.
         /// </summary>
         /// <param name="loggingBuilder">The builder containing the <see cref="ILoggerProvider"/> registrations.</param>
+        [Obsolete("Calling this method causes issues with correctly writing log-information to Application Insights")]
```

Comment:
Maybe something about not using it? 'Call this method on your own risk...' or something.

Reply:
Yes, that would be good.   I added it.